Changes.md

```diff
@@ -6,6 +6,8 @@ Improvements
 
 - RenderMan : Stylized Looks no longer require manual AOV setup. The relevant AOVs are added automatically whenever a stylized display filter is present.
 - Arnold : Added translation of UsdLux IES parameters to Arnold.
+- VisualiserTool : Added new visualisation for orientation (Quatf) data.
+- PrimitiveInspector : Changed order of columns for `quatf` and `quatd` data to show the `W` (real) component first to match the order when dragging to the Python Editor.
```

Comment:
I'm a little confused by the "to match the order when dragging to the Python Editor" here as it reads like it is referring to dragging from the Primitive Inspector. Do we need to clarify this by mentioning dragging from the Visualiser Tool or removing this part entirely?

Reply:
I'd suggest this :

> Changed column order for quaternions to match Imath's conventions.

Reply:
I think I had my brain half in `PrimitiveInspector` land and half in `VisualiserTool` land when I wrote that. Fixed and squashed down.